src/NServiceBus.Core/InitializableEndpoint.cs

```diff
@@ -65,9 +65,10 @@ public InitializableEndpoint(SettingsHolder settings, IContainer container, List
 
             container.ConfigureComponent(b => settings.Get<Notifications>(), DependencyLifecycle.SingleInstance);
 
-            await RunInstallers(concreteTypes).ConfigureAwait(false);
+            var receiveInfrastructure = transportInfrastructure.ConfigureReceiveInfrastructure();
```

Comment:
Good for me too. Although I think strictly speaking creating the infrastructure should not have sideffects I agree it is better to preserve the previous behavior